Description: Bump unicode version to 15
Author: Xavier Guimard <yadd@debian.org>, Julien Puydt <jpuydt@debian.org>
Forwarded: Debian-specific
Last-Update: 2021-01-24

--- a/tools/generate-identifier-regex.js
+++ b/tools/generate-identifier-regex.js
@@ -3,7 +3,7 @@
 var regenerate = require('regenerate');
 
 // Which Unicode version should be used?
-var version = '11.0.0'; // note: also update `package.json` when this changes
+var version = '15.0.0'; // note: also update `package.json` when this changes
 
 // Shorthand function
 var get = function(what) {
