Description: Bump unicode version to 14
Author: Xavier Guimard <yadd@debian.org>, Julien Puydt <jpuydt@debian.org>
Forwarded: Debian-specific
Last-Update: 2021-01-24

--- a/package.json
+++ b/package.json
@@ -24,6 +24,6 @@
   },
   "devDependencies": {
     "regenerate": "~1.0.1",
-    "unicode-11.0.0": "^0.7.8"
+    "unicode-14.0.0": "^0.7.8"
   }
 }
--- a/tools/generate-identifier-regex.js
+++ b/tools/generate-identifier-regex.js
@@ -3,7 +3,7 @@
 var regenerate = require('regenerate');
 
 // Which Unicode version should be used?
-var version = '11.0.0'; // note: also update `package.json` when this changes
+var version = '14.0.0'; // note: also update `package.json` when this changes
 
 // Shorthand function
 var get = function(what) {
